tests/unit/test_loaders.py

```diff
@@ -210,6 +240,31 @@ def test_create_loader_parses_data_path(self):
         self.assertIn('bar', loader.search_paths)
         self.assertIn('baz', loader.search_paths)
 
+    def test_is_builtin_path(self):
+        loader = Loader()
+        path_in_builtins = os.path.join(loader.BUILTIN_DATA_PATH, "foo.txt")
+        path_elsewhere = __file__
+        self.assertTrue(loader.is_builtin_path(path_in_builtins))
+        self.assertFalse(loader.is_builtin_path(path_elsewhere))
+
+    @pytest.mark.skipif(
+        sys.platform == 'win32',
+        reason=(
+            'os.symlink() requires developer mode or elevated permissions'
+        ),
+    )
```

Comment:
i think you said this was just effecting python 3.7 right? if so we should just skip tests for that version

Reply:
Creating symlinks on Windows requires one of the following two things:
 
* Run the script elevated as administrator. This is a temporary change, achieved for example by opening a command line window with right click -> run as administrator.
* Have developer mode enabled. This is a permanent setting admin-level users can make for their account in the Windows settings.

This applies to all languages and therefore all versions of Python. You can read more about this here: https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/

What I have previously mentioned that only affects Windows 3.7 is that `os.realpath()` does not resolve symlinks to their real path, at least not in my testing. This slightly reduces the functionality of the `is_builtin_path()` function for users of Python 3.7 on Windows. If a user on this version and platform decides to symlink the builtin data file inot their `AWS_DATA_PATH`, then it will not be recognized as builtin.